### ISSUE TITLE
Feeds with different query parameters are not considered unique in local plugin

### ISSUE THREAD

**Issue by nagisa** [jangernert/FeedReader#883]
Adding certain feeds results in no output whatsoever. The stderr outputs 

```
feedreader-INFO: 20:05:58.409: addFeed: ID = cedb741a-c565-4f0d-8c0c-23fcc7cc0af8
```

as the only indication an addition was attempted. The UI pops up a bubble that has no content:

![tmp](https://user-images.githubusercontent.com/679122/52969428-09ecf000-33b9-11e9-9d23-2c4b66bc2eef.png)

**Comment by nagisa** [comment]
To reproduce try:

1. Add `https://yourdomain.com/feed?banana`, and notice it gets added.
2. Add `https://yourdomain.com/feed?peach`, and notice it does not get added.

**Comment by brendanlong** [comment]
Example:

1. Add <https://www.brendanlong.com/feeds/all.atom.xml>
2. Add <https://www.brendanlong.com/feeds/all.atom.xml?test>

The second one should do nothing, or pop up an empty box

**Comment by brendanlong** [comment]
@nagisa Can you give an real working feed as an example? In the case I gave above, we will correctly error out if you add multiple because these are all the same feed (as defined by `<link href="https://www.brendanlong.com/feeds/all.atom.xml" rel="self"/>`).

**Comment by nagisa** [comment]
I encountered this issue with feeds from [mangadex](https://mangadex.org/). I cannot link to one, because those are account-specific feeds, but checking the xml for feeds, it is indeed the case that all the feeds do have a matching self link:

```
<atom:link href="https://mangadex.org/rss/<apikey>" rel="self" type="application/rss+xml" />
```

whereas all rss feeds have links that look like:

```
https://mangadex.org/rss/<apikey>/manga_id/<id>
```

One could argue that "they" are in the wrong, however it is not always possible to get "them" to fix it. With that in mind I argue that feedreader should not do any duplication checking based on this tag.

Interestingly, importing the feeds via OPML did not encounter any issues in that regard – the uniqueness checking is only done when adding a single feed.

**Comment by brendanlong** [comment]
I changed the logic in ef4a9c618c287af7ea87f6eba14a9e0b2cedc7ee to just use the link we're given instead of looking at the one in the RSS feed. This should be available in the v2.9.0 release
